src/Host/Broker/Impl/About/AboutController.cs

```diff
@@ -0,0 +1,57 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using System;
+using System.Linq;
+using System.Management;
+using System.Reflection;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.R.Host.Broker.Interpreters;
+using Microsoft.R.Host.Broker.Security;
+using Microsoft.R.Host.Protocol;
+using Microsoft.R.Interpreters;
+
+namespace Microsoft.R.Host.Broker.About {
+    [Authorize(Policy = Policies.RUser)]
+    [Route("/about")]
+    public class AboutController : Controller {
+        private readonly InterpreterManager _interpManager;
+
+        public AboutController(InterpreterManager interpManager) {
+            _interpManager = interpManager;
+        }
+
+        [HttpGet]
+        public AboutHost Get() {
+            var a = new AboutHost();
+
+            a.Version = Assembly.GetExecutingAssembly().GetName().Version;
+            a.OS = Environment.OSVersion;
+            a.Is64BitOperatingSystem = Environment.Is64BitOperatingSystem;
+            a.Is64BitProcess = Environment.Is64BitProcess;
+            a.ProcessorCount = Environment.ProcessorCount;
+            a.WorkingSet = Environment.WorkingSet;
+
+            var query = new SelectQuery(@"Select * from Win32_OperatingSystem");
+            using (var search = new ManagementObjectSearcher(query)) {
+                foreach (var mo in search.Get()) {
+                    a.TotalVirtualMemory = GetValue(mo, "TotalVirtualMemorySize");
+                    a.FreeVirtualMemory = GetValue(mo, "FreeVirtualMemory");
+                    a.TotalPhysicalMemory = GetValue(mo, "TotalVisibleMemorySize");
+                    a.FreePhysicalMemory = GetValue(mo, "FreePhysicalMemory");
+                    break;
+                }
+            }
+
+            a.Interpreters = new RInstallation().GetCompatibleEngines().Select(e => e.Name).ToArray();
```

Comment:
They are two different things, strictly speaking. This one returns engines that are detected (but which may not be configured for use by this broker). `InterpretersControllers` returns the engines that were configured for use by the broker, which may or may not include auto-detected ones, and may also include some extra manually configured ones.

So the question is whether we do need the list of detected interpreters, if the broker wasn't actually configured to use them. If we do, this is useful. I'm just not sure whether we do.